src/components/NavBar.js

```diff
@@ -1,31 +1,63 @@
-import React from "react";
-import {BrowserRouter as Router, Link} from "react-router-dom";
+import React, {useState} from "react";
+import {Link} from "react-router-dom";
 import { faCaretDown } from "@fortawesome/free-solid-svg-icons";
+import { faQuestionCircle } from "@fortawesome/free-solid-svg-icons";
+import { faHammer } from "@fortawesome/free-solid-svg-icons";
+import { faTools } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
 import './NavBar.css';
 import OfekLogo from '../assets/images/OfekLogo.png';
 
-const NavBar = () => {
+const NavBar = props => {
+    
+
+    const toggleSideBar = () => {
+        props.MinMaxMain();
+        if(props.mini) document.getElementById("sidebar").style.width = "250px";
```

Comment:
a.    document.getElementById("sidebar").style.width  = props.mini ? "250px" : "85px" ;

b   we don't want to use 'document' in react code, if you desire to change the style of a component do it in the `style` default prop ,
 https://reactjs.org/docs/dom-elements.html#style ( you can read the whole file to understand more about dom manipulation 
or more in general for css in react: https://www.sitepoint.com/react-components-styling-options/

Reply:
changed classes dynamically instead and declared style in css